### DISCUSSION TITLE
文件下载失败，错误信息: 18 Download aborted.

### DISCUSSION THREAD

**Discussion by 123dmsx** [agalwood/Motrix#1635] [Q&A]
![image](https://github.com/agalwood/Motrix/assets/162784494/9de4f50b-de0e-4b22-802e-de9ca242fdc8)
![image](https://github.com/agalwood/Motrix/assets/162784494/4932f1e7-9358-4ba6-98b5-0e7fca35e4e6)
我的软件无法正常下载文件，图中的下载位置有明显错误与我所设定的下载位置不一样，请问怎么解决

**Comment by mo-mojang** [comment]
是种子下载还是普通下载

**Comment by w1a2n3g4b5o** [comment]
从浏览器助手配置的哪里修改位置，下载器上的不准确
